scalyr_agent/tests/syslog_monitor_test.py

```diff
@@ -332,7 +332,10 @@ def test_run_tcp_server(self):
         expected = "TCP TestXX\n"
         self.connect(s, ("localhost", 8514))
         s.sendall(expected.encode("utf-8"))
-        time.sleep(1)
+
+        # NOTE: Code below relies on sleep. Ideally we should reduce reliance on sleep to make it
+        # more robust and less likely to fail depending on timing.
+        time.sleep(2)
```

Comment:
Note, @ArthurKamalov has a PR under review (which I just approved) which should fix this issue the "right way" without needing to increase the sleep.  I think you might want to revert your changes here to avoid conflicting with him.

Reply:
Yep, that was just a quick hack / workaround in the mean time :)